ivy/functional/backends/torch/linear_algebra.py

```diff
@@ -108,7 +108,7 @@ def eigh(
 def eigvalsh(
     x: torch.Tensor, /, *, UPLO: str = "L", out: Optional[torch.Tensor] = None
 ) -> torch.Tensor:
-    return torch.linalg.eigvalsh(x, UPLO=UPLO, out=out)
+    return torch.linalg.eigvalsh(x, UPLO=UPLO)
```

Comment:
ideally we should make use of native out argument support when it's available, are there any issues with the `out` argument?

Reply:
Sorry, I'm not sure why I removed that `out` argument. I added it now, and all the tests are passing.👍🏻

Reply:
awesome, thanks!